chain/epoch_manager/src/validator_selection.rs

```diff
@@ -543,6 +544,18 @@ mod tests {
         let mut counts: [i32; 2] = [0, 0];
         for h in 0..100_000 {
             let bp = epoch_info.sample_block_producer(h);
+            for shard_id in 0..num_shards {
+                let cp = epoch_info.sample_chunk_producer(h, shard_id);
+                if checked_feature!("stable", SynchronizeBlockChunkProduction, PROTOCOL_VERSION)
+                    && !checked_feature!(
+                        "protocol_feature_chunk_only_producers",
+                        ChunkOnlyProducers,
+                        PROTOCOL_VERSION
```

Comment:
why does this check appear in the test? I thought unless otherwise specified, we always test the stable protocol